extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +215,26 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							// Connection string does not include a password, so will prompt user to optionally enter it manually
+							if (connectionString.includes(`Password=${constants.passwordPlaceholder}`)) {
+								let userPassword = await vscode.window.showInputBox({
+									prompt: constants.includePassword,
+									placeHolder: constants.enterPasswordManually,
+									ignoreFocusOut: true,
+									password: true
+								});
+								if (!userPassword) {
+									await vscode.window.showWarningMessage(constants.userPasswordLater);
```

Comment:
Probably shouldn't await here - this isn't a blocking operation and it's not clear the user has to clear the notification to continue on